webapp/channels/src/components/sidebar/invite_members_button.tsx

```diff
@@ -52,11 +38,11 @@ const InviteMembersButton = (props: Props): JSX.Element | null => {
 
     let buttonClass = 'SidebarChannelNavigator__inviteMembersLhsButton';
 
-    if (!props.touchedInviteMembersButton && Number(totalUserCount) <= Constants.USER_LIMIT) {
+    if (!props.touchedInviteMembersButton) {
         buttonClass += ' SidebarChannelNavigator__inviteMembersLhsButton--untouched';
```

Comment:
I wonder if we just want to remove the bolding altogether.

@esethna or @johndavidlugtu not sure if one of you is familiar with this feature that highlights the `Invite Members` button in the sidebar but it could potentially be causing performance issues so we'd like to remove the request that counts users since it's happening quite frequently. Question is, would we rather have it bolded for everyone until they click it once or just leave it unbolded for everyone?

Reply:
@jwilander can you share a screenshot of this button? Just want to make sure we're on the same page

Reply:
![Screenshot 2023-06-05 at 9 07 52 AM](https://github.com/mattermost/mattermost/assets/14115924/dfc002c7-f2e1-47f9-b078-ecbf083e56b2)

@johndavidlugtu It's the last item in the sidebar, this screenshot is with no highlighting

Reply:
@esethna can I proceed with removing the highlighting option?